app/src/main/res/values-cs/strings.xml

```diff
@@ -34,17 +34,17 @@
     <string name="permission_rationale_settings">Nastavení</string>
     <string name="permission_rationale_dismiss">Zrušit</string>
     <string name="welcome_help">Jak to funguje</string>
-    <string name="welcome_activation">Pokra\u010dovat k\u00a0aktivaci</string>
+    <string name="welcome_activation">Pokračovat k aktivaci</string>
     <string name="welcome_description"><![CDATA[Aplikace bude nepřetržitě monitorovat vaše okolí a&nbsp;zaznamenávat všechny ostatní telefony s aplikací eRouška, ke kterým se přiblížíte.<br/><br/>Pokud se u majitele kteréhokoliv z nich potvrdí onemocnění COVID-19, eRouška vyhodnotí, zda se jedná o rizikový kontakt a upozorní vás.<br/><br/>Když se potvrdí nákaza u&nbsp;vás, eRouška upozorní všechny ostatní uživatele aplikace, se s&nbsp;kterými jste se potkali.]]></string>
-    <string name="welcome_title">D\u00edky eRou\u0161ce ochr\u00e1n\u00edte sebe i\u00a0ostatn\u00ed ve sv\u00e9m okol\u00ed</string>
+    <string name="welcome_title">D\u00edky eRou\u0161ce ochr\u00e1n\u00edte sebe i ostatn\u00ed ve sv\u00e9m okol\u00ed</string>
     <string name="contacts_title">Kontakty</string>
     <string name="send_data">Anonymně upozornit ostatní</string>
     <string name="sent">Odesláno</string>
     <string name="share_app_title">Sdílet aplikaci eRouška</string>
     <string name="menu_share_app">Sdílet aplikaci</string>
     <string name="share_app_text">Ahoj, používám aplikaci eRouška. Nainstaluj si ji taky a společně pomožme zastavit šíření onemocnění COVID-19. Aplikace dokáže anonymně a včas upozornit na rizikové setkání s nakaženým uživatelem a doporučit další postup. Najdeš ji na %s</string>
     <string name="delete_registration">Zrušit registraci</string>
-    <string name="about">O\u00a0aplikaci</string>
+    <string name="about">O aplikaci</string>
```

Comment:
Češtinu bych nestahoval z OneSky. Je to naopak source of truth pro OneSky. Z OneSky se stahuje jenom EN a SK.

Reply:
OK